lab_5_scrapper/scrapper.py

```diff
@@ -13,70 +55,128 @@ def __init__(self, path_to_config: Path) -> None:
         """
         Initializes an instance of the Config class
         """
-        pass
+        self.path_to_config = path_to_config
+        self._validate_config_content()
+        self.config_dto = self._extract_config_content()
+        self._seed_urls = self.get_seed_urls()
```

Comment:
it is funny. to set his own attribute, you ask his own getter to provide it. 

getters are needed for the outer world. For internal use, you should access dto directly